testing/profiler-tests/build.gradle

```diff
@@ -4,6 +4,7 @@ plugins {
 
 dependencies {
   testImplementation project(":profiler")
+  testImplementation("org.slf4j:slf4j-simple")
```

Comment:
`profiler-tests` did not output any logs, so I added `slf4j-simple` to make debugging failures easier